venter/tests.py

```diff
@@ -32,17 +32,29 @@ def create_complaint(user, **kwargs):
         create_complaint(self.user.profile)
         create_complaint(get_new_user().profile)
         create_complaint(self.user.profile, status=STATUS_DELETED)
+        create_complaint(self.user.profile, description='garbage')
 
         url = '/api/venter/complaints'
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(len(response.data), 2)
+        self.assertEqual(len(response.data), 3)
 
-        url = '/api/venter/complaints?filter=me'
+        # Tests for Relevant complaints
+        url = '/api/venter/relevant'
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.data), 4)
```

Comment:
How can a deleted complaint be relevant? This api should give 3 complaints as well.